build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -x
+# "docker manifest" requires experimental feature enabled
+export DOCKER_CLI_EXPERIMENTAL=enabled
+
+# get baseimg (ceph/ceph:tag)
+dockerfile="deploy/cephcsi/image/Dockerfile"
+baseimg=$(awk 'NR==1{print $2}' "${dockerfile}")
+
+# get image digest per architecture
+# {
+#   "arch": "amd64",
+#   "digest": "sha256:XXX"
+# }
+# {
+#   "arch": "arm64",
+#   "digest": "sha256:YYY"
+# }
+manifests=$("${CONTAINER_CMD:-docker}" manifest inspect "${baseimg}" | jq '.manifests[] | {arch: .platform.architecture, digest: .digest}')
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
`--privileged` may not always work, only when running as root. This may not be a problem when running in Travis CI though, but on local developers' systems.

Could you document the requirement in the top of this script?

Reply:
am running docker with a normal user, didn't faced any issue